src/components/composite/common/Discussion/PostComment.tsx

```diff
@@ -26,10 +28,13 @@ const PostComment: FC<PostCommentProps> = ({ onPostComment, title, placeholder }
     post: yup.string().required(t('forms.validations.required')),
   });
 
-  const handleSubmit = async (values: formValues, _helpers: FormikHelpers<formValues>) => {
+  const handleSubmit = (values: formValues, _helpers: FormikHelpers<formValues>) => {
     if (onPostComment) {
-      await onPostComment(values.post);
-      _helpers.resetForm();
+      return (onPostComment(values.post) as Promise<FetchResult<void>>)?.then(result => {
```

Comment:
this is a `void | Promise<FetchResult<void>>` casted to just `Promise<FetchResult<void>>`, haven't found a way to make it cleaner... just the `?` doesn't work and gives me the error "Type `void | Promise...` doesn't have the function `then`